### ISSUE TITLE
Fix Python 3.5 environment in Travis CI

### ISSUE THREAD

**Issue by pferate** [google/oauth2client#532]
Currently the build images that Travis uses do not have Python 3.5 installed, so it must be installed afterwards.  Without it installed, tox will fail when it tries to create the py35 environment because it can't find the interpreter.

There are a few ways of handling it, mentioned below, from travis-ci/travis-ci/issues/4794.

> As mentioned in travis-ci/travis-build#521, in order to use 3.5 with tox, Python 3.5 still needs to be installed.
> 
> You can use deadsnakes as @sjakobi mentioned above, or specify
> 
> ```
> python:
>   - 3.5
> ```
> This, as well as using the APT addon, will install Python 3.5 in all of the jobs.
> 
> One can also use a little matrix trick:
> 
> ```
> matrix:
>   include:
>     - python: 3.5
>       env:
>       - TOXENV=py35
> ```
> (Similar trick should work for the APT addon.)

We could also install 3.5 in the `scripts/install.sh` script.

I tried using 3.5 for the main version, which worked for all the environments except for `gae`.

Which would be the preferred way of implementing it, until it's fixed upstream?

**Comment by pferate** [comment]
I'm using the following method for now:

```
# TODO(issue 532): Fix syntax when 3.5 is natively available upstream
matrix:
  include:
    - python: 3.5
      env:
      - TOX_ENV=py35
```

**Comment by jonparrott** [comment]
I'm considering this a non-issue.
